src/__test__/platform.spec.ts

```diff
@@ -14,16 +14,17 @@ describe("platform", () => {
 
     test("importRaw", async () => {
         const privateKey =
-            "a05f81608217738d99da8fd227897b87e8890d3c9159b559c7c8bbd408e5fb6e";
+            "2c459a04538af99de4b5150ccd213392ea683cc33e6e28a4dfbf8d2c5706032a40200442f046f5012df6d6acf8105be68278a6d3bd631603ab86fed89cdf9901";
         const key = await cckey.platform.importRaw({
             privateKey,
             passphrase: "satoshi"
         });
         const publicKey =
-            "0eb7cad828f1b48c97571ac5fde6add42a7f9285a204291cdc2a03007480dc70639d80c57d80ba6bb02fc2237fec1bb357e405e13b7fb8ed4f947fd8f4900abd";
+            "40200442f046f5012df6d6acf8105be68278a6d3bd631603ab86fed89cdf9901";
         expect(key).toBe(keyFromPublicKey(KeyType.Platform, publicKey));
     });
 
+    // FIXME
```

Comment:
If there is meta data as shown below, how can I generate a public key for testing?

Reply:
To generate the encrypted secret, you can utilize the subcommand `account`.  `./target/debug/foundry account --help` will helps

Reply:
I'll check it : )